apps/web/src/components/layout/components/PrivatePageLayout.tsx

```diff
@@ -45,7 +47,22 @@ export function PrivatePageLayout() {
     [environment]
   );
 
-  return (
+  function clerkPrivateWrapper(children: React.ReactNode) {
+    if (!IS_CLERK_AUTH_ENABLED) {
+      return <>children</>;
+    }
+
+    return (
+      <>
+        <SignedIn>{children}</SignedIn>
```

Comment:
Is it possible to add these in the AppRoutes files or does react-router-dom complain about <Route> not being a direct children of <Router/>?